cleanlab/internal/validation.py

```diff
@@ -18,12 +18,18 @@
 Checks to ensure valid inputs for various methods.
 """
 
+from typing import Any, List, Optional
 import warnings
 import numpy as np
 import pandas as pd
 
 
-def assert_valid_inputs(X, y, pred_probs=None, multi_label=False):
+def assert_valid_inputs(
+    X: np.ndarray,
```

Comment:
Let's make this type `Any` because we want to support as many data formats as possible.